overlord/devicestate/users_test.go

```diff
@@ -481,29 +484,20 @@ func (s *usersSuite) TestCreateUserFromAssertionWithForcePasswordChange(c *check
 	user["force-password-change"] = "true"
 	lusers := []map[string]interface{}{user}
 	s.makeSystemUsers(c, lusers)
-
-	// mock the calls that create the user
-	defer devicestate.MockOsutilAddUser(func(username string, opts *osutil.AddUserOptions) error {
-		c.Check(username, check.Equals, "guy")
-		c.Check(opts.Gecos, check.Equals, "foo@bar.com,Boring Guy")
-		c.Check(opts.Sudoer, check.Equals, false)
-		c.Check(opts.Password, check.Equals, "$6$salt$hash")
-		c.Check(opts.ForcePasswordChange, check.Equals, true)
-		return nil
-	})()
```

Comment:
why is this not needed anymore?

Reply:
It wasn't used. `s.createUserFromAssertion` adds the identical check for this. See line 494